picus.rkt

```diff
@@ -1,23 +1,24 @@
 #lang racket
 ; common require
-(require (prefix-in tokamak: "./picus/tokamak.rkt")
-         (prefix-in utils: "./picus/utils.rkt")
+(require (prefix-in utils: "./picus/utils.rkt")
          (prefix-in config: "./picus/config.rkt")
          (prefix-in solver: "./picus/solver.rkt")
          (prefix-in r1cs: "./picus/r1cs/r1cs-grammar.rkt")
          (prefix-in dpvl: "./picus/algorithms/dpvl.rkt")
          (prefix-in pre: "./picus/precondition.rkt")
          "picus/tmpdir.rkt"
          "picus/ansi.rkt"
-         "picus/verbose.rkt"
-         "picus/global-inputs.rkt")
+         "picus/global-inputs.rkt"
+         "picus/logging.rkt"
+         "picus/framework.rkt"
+         "picus/exit.rkt")
 
 ; =====================================
 ; ======== commandline parsing ========
 ; =====================================
 ; parse command line arguments
-(define arg-r1cs #f)
-(define arg-circom #f)
```

Comment:
So these arguments have disappeared?

Reply:
Yeah, we unify the input and infer which format the input is from file extension.